PennyPincher/PennyPincherAndroidGesturesImporter.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  PennyPincherAndroidGesturesImporter.swift
+//  PennyPincher
+//
+//  Created by Raf Cabezas on 1/30/17.
+//
+
+import Foundation
+
+/// PennyPincherAndroidGesturesImporter
+/// Example usage:
+///
+///    for gesture in PennyPincherAndroidGesturesImporter.translatedGestures(fromURL: androidGesturesFileURL) {
+///        if let template = PennyPincher.createTemplate(gesture.id, points: gesture.allPoints) {
+///            pennyPincherGestureRecognizer.templates.append(template)
+///        }
+///    }
+///
+public final class PennyPincherAndroidGesturesImporter {
+    
+    /// Default android gestures file is called 'gestures', no extension
+    public static var defaultImportFileURL: URL? {
+        guard let url = Bundle.main.url(forResource: "gestures", withExtension: nil) else {
+            print("File not found")
+            return nil
+        }
+        
+        return url
+    }
+    
+    public static func translatedGestures(fromURL url: URL, debug: Bool = false) -> [ImportedGesture] {
```

Comment:
Now that it's a URL and not a string, the type really does say everything needed, so you can go back to `from:` :)